test/dsl/dsl_basics.jl

```diff
@@ -110,7 +117,8 @@ let
     @test rn == rn2
 end
 
-@testset "make_reaction_system can be called from another module" begin
```

Comment:
Chanegd the `@testset` block to ` let .. end` block like for all of our other tests. Not sure if there is a reason for this here, since it is all already in a testset?